### ISSUE TITLE
duplicate method issue

### ISSUE THREAD

**Issue by SMoravcik** [open-telemetry/opentelemetry-go#1732]
go get within my app with Redis, now getting the following error

```
# go.opentelemetry.io/otel/trace
../go.opentelemetry.io/otel/trace/config.go:117:2: duplicate method private
```

**Comment by Aneurysm9** [comment]
Please try with Go >= 1.14 as that is the minimum supported Go version for this library.

**Comment by SMoravcik** [comment]
will try, at this moment using the OS available version of 1.13.8

**Comment by MrAlias** [comment]
https://github.com/open-telemetry/opentelemetry-go/issues/1715
